core/src/main/java/org/apache/accumulo/core/iterators/system/LocalityGroupIterator.java

```diff
@@ -96,6 +96,11 @@ public LocalityGroupContext(LocalityGroup[] groups) {
       }
       defaultGroup = foundDefault;
     }
+
+    @Deprecated
+    public LocalityGroupContext(LocalityGroup[] groups) {
```

Comment:
This isn't public API. It's internal only, so if we're not using it internally, we don't need to keep it around deprecated for any user code. It can just be deleted outright.